### PR TITLE
Css fix

### PR DESCRIPTION
**Pull request by PhilippAubert**
Changing a lot of CSS for mobile optimation. 
Also styling the range-input.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -4,18 +4,17 @@
 .App {
   text-align: center;
   display: grid;
-  grid-template-rows: 15vh auto 10vh;
+  grid-template-rows: 10vh auto 10vh;
   height: 100vh;
-  background: #fc0303;
+  background: #0c0d0d;
```
Comment (mpagels) [inline src/App.css:9]: Would it be a good idea to use some css variables to get to know what color this is? Another point for a variable would be, that you could easily use it again and don't try to find the right hex again.

File: src/App.css [hunk 2]
```diff
@@ -4,18 +4,17 @@
...
 }
 
 .App-Header {
   grid-row: 1/2;
+  border-radius: 5px;
   display: flex;
-  justify-content: center;
-  flex-direction: row;
   align-items: center;
-  border-radius: 5px;
```
Comment (mpagels) [inline src/App.css:18]: You deleting `border-radius` here and inserting `border-radius` again. Is this intended? Just pointing at it. Maybe a mistake?

File: src/components/Effects.js [hunk 3]
```diff
@@ -14,22 +16,22 @@ export default function Effects({
 
   return (
     <div className="Function-Board">
-      <div className="Amp-bar">
-        <h2>SET AUDIO EFFECTS</h2>
+      <div className="Effects-Bar">
+        <p className="Effects-Headline">SET AUDIO EFFECTS</p>
       </div>{" "}
-      <h2> Reverb </h2>
+      <p className="Effect-Value"> REVERB </p>
```
Comment (mpagels) [inline src/components/Effects.js:22]: I would recommend to work here a bit semantically. And using a `label` element with the input.

File: src/components/FilterBoard.js [hunk 4]
```diff
@@ -22,18 +22,27 @@ export default function FilterBoard({
...
+          id="highpass"
+          onClick={changeFilterType}
+          className="FilterButton"
+        >
           <p className="Filter-Box"> HP </p>
         </button>
       </div>
 
-      <h2> Cutoff </h2>
+      <p className="Filter-Value"> CUTOFF </p>
```
Comment (mpagels) [inline src/components/FilterBoard.js:43]: Here again I would use a label for the input.

File: src/components/Footer.js [hunk 5]
```diff
@@ -7,9 +7,6 @@ export default function Footer({ onClickSave, onClickLoad }) {
     <div className="Footer">
       <Switch>
         <div className="FooterNav">
-          <Route>
-            <h2>CONFIG</h2>
-          </Route>
```
Comment (mpagels) [inline src/components/Footer.js:12]: I'm just seeing this. Why do you use here `Route`? 
I though we use the `Route` to render some Component, if the path is corresponding to the path in the `Route`
Similiar to this:
```
<Route path="/about">
    <About />
</Route>
```

I think you don't need the `Route` here at all.

File: src/components/Oscillators.js [hunk 6]
```diff
@@ -34,10 +34,9 @@ export default function Oscillators({
 
   return (
     <div className="Function-Board">
-      <h2>SET OSCILLATOR</h2>
-
+      <p>SET OSCILLATOR</p>
       <div className="Vco-bar">
-        <h2>OSC 1</h2>
+        <p>OSC 1</p>
```
Comment (mpagels) [inline src/components/Oscillators.js:39]: Why do you changed this to paragraphs? headlines seems a good choice at first.

File: src/components/css/Effects.css [hunk 7]
```diff
@@ -0,0 +1,8 @@
+.Effect-Value {
+  padding: 0%;
```
Comment (mpagels) [inline src/components/css/Effects.css:2]:
```suggestion
  padding: 0;
```

File: src/components/css/FilterBoard.css [hunk 8]
```diff
@@ -1,16 +1,27 @@
...
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    margin: 0 2px;
-    background: #fc0303;
-  }
+.FilterButton {
+  border-radius: 6px;
+  display: flex;
+  flex-direction: row;
```
Comment (mpagels) [inline src/components/css/FilterBoard.css:4]: You don't need `flex-direction: row`: It's the default value, if you're using `display: flex`.
Saw this on another position as well, but was not part of the pull request. But worth to change.

File: src/components/css/FilterBoard.css [hunk 9]
```diff
@@ -1,16 +1,27 @@
...
+  align-items: center;
+  margin: 0 2px;
+  background: #fc0303;
```
Comment (mpagels) [inline src/components/css/FilterBoard.css:7]: Here again. Maybe worth to use a variable?

**Review by mpagels — CHANGES_REQUESTED** [review]
I just mentioned only two times the `semantic` problem, with the labels for the inputs. I saw it a lot more often in the pull request. Maybe you can take a look inside your code to change this by yourself.

File: src/App.css (hunk 2 above)
Comment (PhilippAubert) [reply src/App.css:18]: Deleted by mistake, yes. Then added back a few minutes later. That was confusing... sorry.

File: src/components/Oscillators.js (hunk 6 above)
Comment (PhilippAubert) [reply src/components/Oscillators.js:39]: As far as I remember, @marw didn't like the <h2>

Comment (PhilippAubert) [reply src/components/Oscillators.js:39]: As far as I remember, @marwinburesch didn't like the h2 ...
